settings.sh

```diff
@@ -1,3 +1,4 @@
 STANDALONE="yes"
 NETWORK="192.168.1"
 LAN_IP="192.168.1.23"
+INTERFACE="" 
```

Comment:
Instead of leaving it empty you can place an example value, (like `eth1`).
You can also use comments to explain what each variable is and how to set its value.
For example the comment here could be:

`### The network interface connected to the LAN of the ltsp clients.`

Fortunately, the comments in Bash and in Ruby are the same.